src/main/java/seedu/address/model/event/DateTime.java

```diff
@@ -12,7 +12,8 @@
  */
 public class DateTime {
     public static final String MESSAGE_CONSTRAINTS =
-            "Date times should follow this format: DD-MM-YYYY HH:mm and only values within valid range are allowed.";
+            "Date times should follow this format: DD-MM-YYYY HH:mm and "
+            + "only values within valid range are allowed (start date time cannot be after the end date time).";
```

Comment:
Will this affect the tests?

Reply:
nope! the tests call `DateTime.MESSAGE_CONSTRAINTS` directly, so no tests are affected by this change :-)